### PR TITLE
Make import_docs.sh work on Mac

### PR DESCRIPTION
**Pull request by erictune**

### CODE REVIEW

File: _tools/import_docs.sh [hunk 1]
```diff
@@ -20,19 +21,20 @@ git rm -rf ${OUTDIR}/docs/ ${OUTDIR}/examples/ > /dev/null
...
+mv $tmpdir/examples ${OUTDIR}
 git add ${OUTDIR}/docs/ ${OUTDIR}/examples/ > /dev/null
-rmdir $TMPDIR
+rmdir $tmpdir
 
 echo stripping
 for dir in docs examples; do
     find ${OUTDIR}/${dir} -type f -name \*.md | while read X; do
-        sed -i \
+        sed -i .import_docs_tmp \
```
Comment (nikhiljindal) [inline _tools/import_docs.sh]: I guess, we need to update https://github.com/GoogleCloudPlatform/kubernetes/blob/master/build/versionize-docs.sh#L56 as well, then

**Comment by thockin** [comment]
This just adds a suffix to tempfiles - what was going wrong?

LGTM

**Comment by erictune** [comment]
sed on BSD requires an argument after `-i`.

File: _tools/import_docs.sh (hunk 1 above)
Comment (erictune) [inline _tools/import_docs.sh]: Thanks.  I'll fix that when I encounter it.

Comment (thockin) [inline _tools/import_docs.sh]: Ahh, in versionize we actually call `gsed` on macOS - is that a better fix?  The sed logic here and there really should be ~identical.

**Pull request merged**